common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -177,6 +199,39 @@ def _check_provider_configurations(self):
 
         return metrics
 
+    def _check_no_config(self, provider_config, provider_info, null_config_count):
+        """Helper to check providers with no direct SAML configuration."""
+        try:
+            default_config = SAMLConfiguration.current(provider_config.site_id, 'default')
+            if not default_config or default_config.id is None:
+                # Resolution: Create a SAML configuration for this provider
+                # or create a default configuration for the site
+                self.stdout.write(
+                    f"[WARNING] {provider_info} has no direct SAML configuration and "
+                    "no matching default configuration was found."
+                )
+                null_config_count += 1
+                return null_config_count
+
+            if not default_config.enabled:
+                # Resolution: Enable the default SAML configuration
+                # or create a specific configuration for this provider
+                self.stdout.write(
+                    f"[WARNING] {provider_info} has no direct SAML configuration and "
+                    f"the default configuration (id={default_config.id}, enabled=False)."
+                )
+                null_config_count += 1
+        except SAMLConfiguration.DoesNotExist:
+            # Resolution: Create a SAML configuration for this provider
+            # or create a default configuration for the site
```

Comment:
```suggestion
            # Resolution: Link this provider with a SAML configuration
            # or link it with a default configuration for the site
```

We can change it to this.